staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -131,7 +131,7 @@ func dataHandler(ctx context.Context, dev *driver.CustomizedDev) {
 			ObservedDesired: twin.ObservedDesired,
 			VisitorConfig:   &visitorConfig,
 			Topic:           fmt.Sprintf(common.TopicTwinUpdate, dev.Instance.ID),
-			CollectCycle:    time.Duration(twin.Property.CollectCycle),
+			CollectCycle:    time.Millisecond * time.Duration(twin.Property.CollectCycle),
```

Comment:
The reportcycle in this file need to be fixed (like in **pushHandler** function)